README.md

```diff
@@ -1,10 +1,7 @@
-# Consul
-
-<p>
-  <a href="https://consul.io" title="Consul website">
-    <img src="./website/public/img/logo-hashicorp.svg" alt="HashiCorp Consul logo" width="200px">
-  </a>
-</p>
+<h1>
+  <img src="./website/public/img/logo.svg" align="left" height="46px" alt="Consul logo"/>
```

Comment:
We decided not to use the Github magic `#gh-(light|dark)-mode-only` string solution built for this purpose because it would not be understood by other systems that may render this README, such as https://pkg.go.dev/